content/en/docs/Endpoints/getTranscodeDecision.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "getTranscodeDecision"
+linkTitle: "getTranscodeDecision"
+categories:
+- Transcoding
+OpenSubsonic:
+- Extension
+description: >
+  Returns a transcode decision for a given media file.
+---
+
+`http://your-server/rest/getTranscodeDecision`
+
+Returns a transcode decision for a given media file. This endpoint is used by clients to determine if a media file should be transcoded before streaming. The server will analyze the provided client information and return a decision.
+This endpoint must be accessed using a POST request
+
+### Parameters
+
+| Parameter | Req. | OpenS. | Default | Comment |
+| --- | --- | --- | --- | --- |
+| `mediaId` | **Yes** | | | The ID of the media to be transcoded. |
+| `mediaType` | **Yes** | | | Either song or podcast so the server knows what the mediaId is referring to. |
+
+### Request Body
+
+The request body must be a JSON object containing the client's capabilities. See the [`ClientInfo`](../payloads/clientinfo) documentation for more details.
+
+{{< tabpane persist=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="ClientInfo" lang="json">}}
+{
+  "name": "Play:1",
+  "platform": "Sonos",
+  "maxAudioBitrate": 512000,
+  "maxTranscodingAudioBitrate": 256000,
+  "directPlayProfiles": [
+    {
+      "containers": [ "mp3" ],
+      "audioCodecs": [ "mp3" ],
+      "protocols": [ "http" ],
+      "maxAudioChannels": 2
+    },
+    {
+      "containers": [ "flac" ],
+      "audioCodecs": [ "flac" ],
+      "protocols": [],
+      "maxAudioChannels": 2
+    }
+    ,
+    {
+      "containers": [ "mp4" ],
+      "audioCodecs": [ "flac", "aac", "alac" ],
+      "protocols": [],
+      "maxAudioChannels": 2
+    }
+    ],
+    "transcodingProfiles": [
+    {
+      "container": "mp3",
+      "audioCodec": "mp3",
+      "protocol: "http",
```

Comment:
There is a closing quotation mark missing here, causing the rest of the example to be highlighted as an error on the web UI.